README.md

```diff
@@ -184,10 +184,24 @@ RuntimeError: Calculated padded input size per channel: <smaller than 5 x 5>.
 Kernel size: (5 x 5). Kernel size can't be greater than actual input size
 ```
 
-#### Testing
+#### Inference
 The [`testing/main.py`](src/gz21_ocean_momentum/testing/main.py) script runs the
-model testing stage. You select a trained model and a region (which should be
-new/unseen) to test it on.
+model testing stage. This consists of running a trained model on a dataset. 
+The model's output are then stored as an artefact. This step
+should ideally be run with a GPU device available.
+
+Specifically, one can run the inference step by interactively
+running the following from the src/gz21_ocean_momentum directory:
+
+'python -m testing.main --n_splits=40'
```

Comment:
Add explanation what the expected output is and what the parameter n_splits does (and potential other parameters)

Reply:
The dataset is split in n_splits chunks, each one of those is processed and the final output dataset is obtained by "collating" the output datasets. I believe I had done this due to memory issues.